src/features/location/components/LocationSearchWidget.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { SearchLocationModal } from 'features/location/components/SearchLocationModal'
+import { getLocationTitle } from 'features/location/helpers/getLocationTitle'
+import { VenueModal } from 'features/search/pages/modals/VenueModal/VenueModal'
+import { useLocation } from 'libs/geolocation'
+import { useModal } from 'ui/components/modals/useModal'
+import { VerticalSeparator } from 'ui/components/Separator'
+import { TouchableOpacity } from 'ui/components/TouchableOpacity'
+import { BicolorLocationPointer } from 'ui/svg/icons/BicolorLocationPointer'
+import { LocationPointer } from 'ui/svg/icons/LocationPointer'
+import { Spacer, Typo } from 'ui/theme'
+
+export const LocationSearchWidget = () => {
+  const { isGeolocated, isCustomPosition, userPosition, place } = useLocation()
+
+  const locationTitle = getLocationTitle(place, userPosition)
+
+  const {
+    visible: locationModalVisible,
+    showModal: showLocationModal,
+    hideModal: hideLocationModal,
+  } = useModal()
+
+  const {
+    visible: venueModalVisible,
+    showModal: showVenueModal,
+    hideModal: hideVenueModal,
+  } = useModal()
+
+  const isWidgetHighlighted = isGeolocated || !!isCustomPosition
+
+  return (
+    <Container>
+      <VerticalSeparator />
+      <Spacer.Row numberOfSpaces={2} />
+      <LocationButton onPress={showLocationModal} testID="Ouvrir la modale de localisation">
+        {isWidgetHighlighted ? <LocationPointerFilled /> : <LocationPointerNotFilled />}
+        <Spacer.Row numberOfSpaces={1} />
+        <LocationTitle>{locationTitle}</LocationTitle>
+      </LocationButton>
+      <React.Fragment>
+        <VenueModal visible={venueModalVisible} dismissModal={hideVenueModal} />
+        <SearchLocationModal
+          visible={locationModalVisible}
+          dismissModal={hideLocationModal}
+          showVenueModal={showVenueModal}
+        />
+      </React.Fragment>
+    </Container>
+  )
+}
+const Container = styled.View({
+  flexDirection: 'row',
+  height: 32,
```

Comment:
toutes nos tailles sont des multiples de 4

pour éviter les erreurs, on utilise la fonction `getSpacing`

```suggestion
  height: getSpacing(8),
```